### ISSUE TITLE
Explicit no release

### ISSUE THREAD

**Issue by oulenz** [autopub/autopub#18]
Apologies if I missed something, but there is no explicit option in the RELEASE.md file for not triggering a release, right?

(Actually, it seems that by not writing one of the three release types, a release is averted, but that results in an error, so it is not very graceful.)

The motivation is documenting changes in the changelog.

Maybe it would be sufficient to update the README of this project to make it clear that this is possible already.

**Comment by marcoacierno** [comment]
If you don't create a RELEASE.md CI should pass doing nothing, for example here

Run with no release file: https://github.com/strawberry-graphql/strawberry/runs/2629096433
Run with release: https://github.com/strawberry-graphql/strawberry/runs/2629084517

This is the workflow file: https://github.com/strawberry-graphql/strawberry/blob/main/.github/workflows/release.yml

@oulenz if you can share the CI you are using and your file it would be useful to debug your issue with CI failing when there is no file :)

**Comment by justinmayer** [comment]
@marcoacierno: I believe Oliver is referring to being able to use the `RELEASE.md` file to add change-log entries to `CHANGELOG.md` *without* triggering a release.

Oliver said above that this use case is apparently (and quite accidentally) achievable by not specifying one of the three expected release types. In that case, the entry in the `RELEASE.md` file is indeed appended to the change-log, at the cost of an unhandled error.

I think it's safe to say that AutoPub's original design assumes that the existence of a `RELEASE.md` file conveys the intention to issue a new release. Upon first reading this issue, I wasn't sure how I felt about hijacking that process to add a change-log entry *without* issuing a new release, particularly if we were to sanction that behavior with official documentation. But the more that I think about it, I do understand (and have myself experienced) that there are times in which it does not make sense to issue a release right away, and then one must remember to collate previous commit message bits into the `RELEASE.md` file when the time finally comes to issue a release. The flow that Oliver proposed would allow folks to add change-log entries incrementally and synchronously, as features and bug-fixes are added, as opposed to collecting them at release time.

That said, in a situation where a release is *not* being issued, no version/date header should be appended, which I imagine would require changes to AutoPub's process for updating the change-log.

In short, I am open to the idea of sanctioning this behavior, but I think someone would need to make a few corresponding changes to eliminate the error and ensure it updates the change-log in an appropriate manner.
